config/settings.py

```diff
@@ -214,6 +216,26 @@
 PASSWORD_RESET_TIME_LIMIT_MINUTES = 180
 ACCOUNT_DELETE_TIME_LIMIT_MINUTES = 180
 
+
+# Default dummy settings, please override in local_settings.py
+DEFAULT_FROM_EMAIL = "testing@example.com"
+SPARKPOST_RELAY_DOMAIN = 'replies.karrot.localhost'
+HOSTNAME = 'https://localhost:8000'
+SITE_NAME = 'karrot.localhost'
```

Comment:
What `SITE_NAME` and `DEFAULT_FROM_EMAIL` should contain is a bit unclear right now, but will be addressed in #490 ... can get validated in the check command, etc.